# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/spirv-cross-internal/Cargo.toml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/spirv-cross-internal/Cargo.toml b/third_party/rust/spirv-cross-internal/Cargo.toml
deleted file mode 100644
--- a/third_party/rust/spirv-cross-internal/Cargo.toml
+++ /dev/null
@@ -1,26 +0,0 @@
-# the name has to not be a prefix to any modules!
-[package]
-name = "spirv-cross-internal"
-version = "0.1.0"
-authors = ["Joshua Groves <josh@joshgroves.com>"]
-description = "Safe wrapper around SPIRV-Cross"
-license = "MIT/Apache-2.0"
-homepage = "https://github.com/grovesNL/spirv_cross"
-repository = "https://github.com/grovesNL/spirv_cross"
-readme = "../README.md"
-keywords = ["spirv", "cross"]
-build = "build.rs"
-edition = "2018"
-
-[features]
-default = []
-glsl = []
-hlsl = []
-msl = []
-
-[target.'cfg(not(target_arch = "wasm32"))'.build-dependencies]
-cc = { version = "1", features = ["parallel"] }
-
-[target.wasm32-unknown-unknown.dependencies]
-#wasm-bindgen = "0.2.33"
-#js-sys = "0.3.10"